### ISSUE TITLE
Inserting a link doesn't work with speech recognition software

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6011]
Breaking this down from Eric Wright (@ewaccess) testing session with Dragon NaturallySpeaking (speech recognition software). Please refer to the videorecorded session posted here:
https://make.wordpress.org/accessibility/2018/03/28/accessibility-of-gutenberg-the-state-of-play/

Please refer to the video above, starting from about min. 23:40 

When inserting a link and the link field is focused, voicing the command "Press enter key" triggers browser validation, as if the field was empty. Seems this is not the only case when using speech recognition software, the UI doesn't get an input field is actually populated with some text. See also #6010 and #5468.

**Comment by afercia** [comment]
See also #5468 when this issue was first noticed on the Title field.

**Comment by afercia** [comment]
Removing from the Merge Proposal milestone, as right now it's unclear how to solve this and it should be solved together with the similar issues anyways.

**Comment by paaljoachim** [comment]
@joedolson Joe. Is this issue still valid?

**Comment by annezazu** [comment]
Closing this out as we haven't had an update in three years on a six year old issue and the link editing experience has greatly changed. As of now, I think this issue takes precedence on the experience: https://github.com/WordPress/gutenberg/issues/60901

If that's wrong, let's reopen!
